portal/server/types/banks.ts

```diff
@@ -0,0 +1,22 @@
+type ReportPeriodSchedule = {
+  startMonth: number;
+  endMonth: number;
+}
```

Comment:
This type is going to change in the new update, so could we replace it with this:
```
type BankReportPeriodSchedule = {
  startMonth: number;
  endMonth: number;
}[];
```